lib/makeCmdTasks.js

```diff
@@ -31,20 +32,44 @@ export const makeCmdTasks = async ({ commands, cwd, files, gitDir, shell, verbos
     const resolvedArray = Array.isArray(resolved) ? resolved : [resolved] // Wrap non-array command as array
 
     for (const command of resolvedArray) {
-      // If the function linter didn't return string | string[]  it won't work
-      // Do the validation here instead of `validateConfig` to skip evaluating the function multiple times
-      if (isFn && typeof command !== 'string') {
-        throw new Error(
-          configurationError(
-            '[Function]',
-            'Function task should return a string or an array of strings',
-            resolved
+      if (isFn && command && typeof command === 'object') {
+        if (
```

Comment:
Can we flip the condition and do an early return?

Reply:
done